test/unit/radio-helpers.spec.js

```diff
@@ -0,0 +1,84 @@
+describe('Marionette radio helpers', function() {
+
+  describe('when creating a Marionette.Object', function() {
+
+    beforeEach(function() {
+      this.clickStub1 = this.sinon.stub();
+      this.clickStub2 = this.sinon.stub();
+      this.clickStub3 = this.sinon.stub();
+      this.Object = Marionette.Object.extend({
+        radioEvents: {
+          'foo bar' : this.clickStub1
```

Comment:
This is weird because you run into the `object2` problem w/ prototypes.

The easier thing is

```js
```
O = extend({
  radioEvents: {'foo bar': 'barHandler'}

  barHandler: function() {}
});

o = new O();
o.barHandler = this.barHandler;
```